src/components/core/Button/Button.js

```diff
@@ -18,10 +18,11 @@ const Button = styled.button`
 	color: ${props => props.theme.colors[getColor(props.variant)]};
 	background-color: ${props =>
 		props.theme.colors[getBgColor(props.variant)] || 'transparent'};
+	text-decoration: none;
 	user-select: none;
 
-	&:hover:enabled,
-	&:active:enabled {
+	&:hover:not(:disabled),
```

Comment:
What's the difference? Semantically seems the same so why doing this change?

Reply:
As I remember it's to use a Button with `<a>`, that doesn't have `:disabled` and `:enabled`. I can add a comment.

Reply:
Yep would be great to have an explanation. Thanks!